source/_integrations/atag.markdown

```diff
@@ -0,0 +1,79 @@
+---
+title: Atag
+description: Instructions on how to setup Atag integration.
+logo: atag.png
+ha_category:
+  - Climate
+  - Water heater
+  - Sensor
+ha_release: 0.105
+ha_iot_class: Local Polling
+---
+
+The `Atag` integration allows Home Assistant to connect to [Atag One](https://atag-one.com) thermostats, reporting and setting its status.
+The integration implements the following platforms:
+
+- Climate
+- Water Heater
+- Sensor
+
+## Configuration
+
+The Atag integration can be enabled directly from Home Assistant. Navigate to `configuration`, then `integrations` and click `add`. Click `Atag` to initiate the configuration. Only the IP Address has to be provided, but be sure to add your email address if you experience connectivity issues.
+
+{% configuration %}
+host:
+  description: Atag hostname or IP address.
+  required: true
+  type: string
+email:
+  description: Email registered in the Atag App.
+  required: false
+  type: string
+port:
+  description: Port to reach the Atag API. Only needed if connecting through alternative routes.
+  required: false
+  type: integer
+{% endconfiguration %}
+
+## Climate
+
+The `Atag` climate platform provides current and target temperature information for the heating system, boiler status and HVAC mode.
+
+### Component services
```

Comment:
We don't use the term "component" when user-facing content. Use "Integration" instead.
```suggestion
### Integration services
```